antarest/study/service.py

```diff
@@ -206,6 +224,54 @@ def assert_raw(study: Study) -> RawStudy:
     return study
 
 
+class StudySettingsDTO(AntaresBaseModel):
+    time_series: TimeSeriesConfiguration
+    general: GeneralConfig
+    advanced_parameters: AdvancedParameters
+    adequacy_patch: AdequacyPatchParameters
+    optimization: OptimizationPreferences
+    thematic_trimming: ThematicTrimming
+    playlist: Playlist
+
+
+class StudyXpansionDTO(AntaresBaseModel):
+    settings: XpansionSettings
+    candidates: list[XpansionCandidate]
+    constraint: bytes
+
+
+class StudyShortTermStorageDTO(AntaresBaseModel):
+    cluster: STStorage
+    constraints: list[STStorageAdditionalConstraint]
+
+
+class StudyAreasDTO(AntaresBaseModel):
+    id: str
+    properties: AreaProperties
+    ui: dict[str, Any]
+    thermals: list[ThermalCluster]
+    renewables: list[RenewableCluster]
+    st_storages: list[StudyShortTermStorageDTO]
+    hydro: HydroProperties
+
+
+class StudyDataDTO(AntaresBaseModel):
+    """
+    DTO representing data of the whole study.
+    """
+
+    name: str
+    version: StudyVersionStr
+    path: str | None
+
+    areas: list[StudyAreasDTO]
+    links: list[Link]
+    binding_constraints: list[BindingConstraint]
+    settings: StudySettingsDTO
+    xpansion: StudyXpansionDTO | None
+    outputs: dict[str, Simulation]
```

Comment:
For consistency sake we could use a list here too, seem the key is redundant with the name inside `Simulation`